src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+
+### Sidecar JSON (`*_motion.json`)
+
+Generic fields (shared with other BIDS modalities) MUST be present:
+
+| **Key name** | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                   |
+| ------------ | --------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TaskName     | REQUIRED              | string        | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this TaskName field by removing all non-alphanumeric `[a-zA-Z0-9]` characters. |
+
+Generic fields (shared with other BIDS modalities) SHOULD be present: For consistency between studies and institutions, we
+encourage users to extract the values of these fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+| **Key name**                | **Requirement level** | **Data type** | **Description**                                                                                     |
+| --------------------------- | --------------------- | ------------- | --------------------------------------------------------------------------------------------------- |
+| InstitutionAddress          | RECOMMENDED           | string        | The address of the institution in charge of the equipment that produced the composite instances.    |
+| InstitutionalDepartmentName | RECOMMENDED           | string        | The department in the institution in charge of the equipment that produced the composite instances. |
+| InstitutionName             | RECOMMENDED           | string        | The name of the institution in charge of the equipment that produced the composite instances.       |
+| Instructions                | RECOMMENDED           | string        | Text of the instructions given to participants before the recording.                                |
+| TaskDescription             | RECOMMENDED           | string        | Longer description of the task.                                                                     |
+
+Specific MOTION fields MUST be present:
+
+| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                       |
+| ------------------ | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------- |
+| SamplingFrequency  | REQUIRED              | number        | Nominal sampling rate of the tracking system in Hz.                                                                   |
```

Comment:
I would recommend having this `SamplingFrequency` close to `SamplingFrequencyEffective` so it facilitates clarification between both. Even maybe adding some text to explain this difference, since I think this is the first time this kind of distinction is made for SF

Reply:
I agree, we included this as it can be used in virtual environments (for example VR). But I am happy to make this BIDS units complient.